lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -324,9 +324,7 @@ class TapTargets extends Gatherer {
       ${pageFunctions.getNodePathString};
       ${pageFunctions.getNodeSelectorString};
       ${gatherTapTargets.toString()};
-      
```

Comment:
can you keep the line break here but delete the trailing whitespace? It definitely helps with readability (the other line can definitely go)